api/controllers/CommunityController.js

```diff
@@ -55,6 +61,33 @@ module.exports = {
     .catch(res.serverError)
   },
 
+  addSlack: function (req, res) {
+    var code = req.query.code,
+      redirect_uri = req.protocol + '://' + process.env.DOMAIN + req.path,
+      url = format('%s?client_id=%s&client_secret=%s&code=%s&redirect_uri=%s',
+                       slackAuthAccess,
+                       process.env.SLACK_APP_CLIENT_ID,
+                       process.env.SLACK_APP_CLIENT_SECRET,
+                       code,
+                       redirect_uri);
+
+    get(url).spread((resp, body) => {
+      var parsed = JSON.parse(body);
+      Community.find(req.param('communityId')).then(function (community) {
+        var communityToUpdate = new Community({id: req.param('communityId')})
+        communityToUpdate.save({
```

Comment:
you can call `save` on the instance of `community` you get from `find`; no need to do `new Community()` here